### DISCUSSION TITLE
Where should Snyk security issue be reported?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [handlebars-lang/handlebars.js#1907] [Q&A]
Snyk reports [this](https://cwe.mitre.org/data/definitions/1321.html) vulnerability introduced by handlebars.

Where should this be reported?

<img width="1495" alt="Screenshot 2022-11-16 at 12 58 01" src="https://user-images.githubusercontent.com/9967422/202178792-bad12fd4-ae49-4f94-983d-72ecd845b3c3.png">

**Comment by jaylinski** [comment]
Handlebars 4.x allows `uglify-js@^3.1.4`, which includes the patched version of `uglify-js` without vulnerabilities.

Is this report from a private project?

**Reply by alexbjorlig** [reply]
Yes - I will try update dependencies to see if the issue goes away 😅

**Comment by my-development** [comment]
@alexbjorlig Hey Alex, we have a similar vulnerability issue from our project, I'm curious if the update fixed the issue from your project.   Thanks!
